PORT 7058 add docs for new aws ocean exporter (#1282)
port-labs/port-docs: docs/build-your-software-catalog/sync-data-to-catalog/cloud-providers/aws/aws.md

# AWS

> :::warning

Reviewer: ```suggestion
:::warning Ocean integration
```

> This documentation refers to the AWS Ocean integration. If you are looking for the AWS Exporter (Lambda), please refer to the [AWS Exporter documentation](./aws-exporter/).

Reviewer: ```suggestion
This documentation refers to the newer AWS Ocean integration. If you are using the older AWS Exporter (Lambda), please refer to the [AWS Exporter documentation](./aws-exporter/).
```

> This documentation refers to the newer AWS Ocean integration. If you are using the older AWS Exporter (Lambda), please refer to the [AWS Exporter documentation](./aws-exporter/).

Reviewer: ```suggestion
This documentation refers to the newer (and recommended) AWS Ocean integration.  
If you are using the older AWS Exporter (Lambda), please refer to the [AWS Exporter documentation](./aws-exporter/).
```

:::

> Our AWS integration allows you to import your AWS resources into Port, according to your configuration.  

Reviewer: ```suggestion
Port's AWS integration allows you to import your AWS resources into Port, according to your configuration.  
```
